Tools-Override/crossgen.sh

```diff
@@ -2,7 +2,9 @@
 
 # Restores crossgen and runs it on all tools components.
 
-__CoreClrVersion=1.1.0
+__CoreClrVersion=2.0.0-preview1-25204-02
```

Comment:
> If crossgen is included with the shared framework though (which I think is the plan?).

It should now be in the the tools folder of the Microsoft.NETCore.App package (so you don't need to restore the Microsoft.NETCore.Runtime.CoreCLR package) but the current plan is to not include it as an exe in the shared framework on disk.

/cc @gkhanna79